tools/migrator/src/fixture.ts

```diff
@@ -33,83 +33,95 @@ export const runFixtures = async () => {
 
   insertedIds.owUser = await db //
     .insertInto("owUser")
+    .onConflict((conflict) => conflict.doNothing())
```

Comment:
very very nitpicky, but can we name this eb to keep consistent with kysely?

Reply:
Why would your fixture loading create conflicts? If the fixtures are not up to date that should be a part of our test-suite?